wiki/bin/update-emails-google

```diff
@@ -54,13 +64,7 @@ for unit in sorted(x, key=sort_in_order):
         print '* %s %s%s' % (r['name']['fullName'], r['primaryEmail'], aliases)
     print
 
-credentials = SignedJwtAssertionCredentials(
-    client_id, private_key,
-    'https://www.googleapis.com/auth/admin.directory.group.readonly',
-    sub=sub_user)
-
-http_auth = credentials.authorize(Http())
-groupadmin = build('admin', 'directory_v1', http=http_auth)
+groupadmin = get_directory()
```

Comment:
If useradmin and groupadmin are now the same, both having both scopes, no need to use two different variables?

Reply:
Done!